Scripts/release.sh

```diff
@@ -0,0 +1,276 @@
+read  -n 1 -p "Has master been merged with devel? [y/n] $cr? " podspec_valid;
+if [ "$podspec_valid" != "y" ]; then
+printf "\nPlease merge devel onto master before proceeding!!\n"
+exit 1
+fi;
+
+# ---- git tag all modules----
+printf "\n\n7. Tagging all modules...\n";
+printf "Tagging core-$OPTIMIZELY_SDK_CORE_VERSION\n";
+git tag -a core-$OPTIMIZELY_SDK_CORE_VERSION -m "Release $OPTIMIZELY_SDK_CORE_VERSION";
+printf "Tagging shared-$OPTIMIZELY_SDK_SHARED_VERSION\n";
+git tag -a shared-$OPTIMIZELY_SDK_SHARED_VERSION -m "Release $OPTIMIZELY_SDK_SHARED_VERSION";
+printf "Tagging datafileManager-$OPTIMIZELY_SDK_DATAFILE_MANAGER_VERSION\n";
+git tag -a datafileManager-$OPTIMIZELY_SDK_DATAFILE_MANAGER_VERSION -m "Release $OPTIMIZELY_SDK_DATAFILE_MANAGER_VERSION";
+printf "Tagging eventDispatcher-$OPTIMIZELY_SDK_EVENT_DISPATCHER_VERSION\n";
+git tag -a eventDispatcher-$OPTIMIZELY_SDK_EVENT_DISPATCHER_VERSION -m "Release $OPTIMIZELY_SDK_EVENT_DISPATCHER_VERSION";
+printf "Tagging userProfile-$OPTIMIZELY_SDK_USER_PROFILE_VERSION\n";
+git tag -a userProfile-$OPTIMIZELY_SDK_USER_PROFILE_VERSION -m "Release $OPTIMIZELY_SDK_USER_PROFILE_VERSION";
+printf "Tagging iOS-$OPTIMIZELY_SDK_iOS_VERSION\n";
+git tag -a iOS-$OPTIMIZELY_SDK_iOS_VERSION -m "Release $OPTIMIZELY_SDK_iOS_VERSION";
+printf "Tagging tvOS-$OPTIMIZELY_SDK_TVOS_VERSION\n";
+git tag -a tvOS-$OPTIMIZELY_SDK_TVOS_VERSION -m "Release $OPTIMIZELY_SDK_TVOS_VERSION"
+printf "\nListing all tags...\n";
+git for-each-ref --count=7 --sort=-taggerdate --format '%(refname)' refs/tags;
+read  -n 1 -p "Are all tags valid? [y/n] $cr? " tagging_valid;
+if [ "$tagging_valid" != "y" ]; then
+    printf "\nCorrect the tag(s) before pushing tags!!\n"
+    exit 1
+fi;
+
+## ---- git push tags ----
+printf "\n\n8. Pushing all git tags...\n"
+git push origin core-$OPTIMIZELY_SDK_CORE_VERSION --verbose;
+printf "\n";
+git push origin shared-$OPTIMIZELY_SDK_SHARED_VERSION --verbose;
+printf "\n";
+git push origin datafileManager-$OPTIMIZELY_SDK_DATAFILE_MANAGER_VERSION --verbose;
+printf "\n";
+git push origin eventDispatcher-$OPTIMIZELY_SDK_EVENT_DISPATCHER_VERSION --verbose;
+printf "\n";
+git push origin userProfile-$OPTIMIZELY_SDK_USER_PROFILE_VERSION --verbose;
+printf "\n";
+git push origin iOS-$OPTIMIZELY_SDK_iOS_VERSION --verbose;
+printf "\n";
+git push origin tvOS-$OPTIMIZELY_SDK_TVOS_VERSION --verbose;
+
+# ---- Verify podspecs ----
+printf "\n"
+read  -n 1 -p "9. Verify all podspecs? Skip if this has already been done. [y/n] $cr? " verify_podspec;
+if [ "$verify_podspec" == "y" ]; then
+    printf "\nVerifying podspecs...\n";
+    pods=(OptimizelySDKCore OptimizelySDKShared OptimizelySDKDatafileManager OptimizelySDKEventDispatcher OptimizelySDKUserProfile OptimizelySDKiOS OptimizelySDKTVOS);
+    number_pods=${#pods[@]};
+
+for (( i = 0; i < ${number_pods}; i++ ));
+do
+    echo "Verifying the ${pods[i]} pod"
+    pod spec lint ${pods[i]}.podspec
```

Comment:
sure...i'll put it after the podspecs are changed.
there's going to be the error: None of your spec sources contain a spec satisfying the dependency because none of the tags are pushed to trunk, but i think we can ignore that and just worry about the other lint errors. ultimately, as long as the version numbers in the podspecs are valid and we push the podspecs to trunk in the correct order, the final podspec lint should pass. the script makes sure both those happen correctly.